cupy/random/_generator_api.pyx

```diff
@@ -174,6 +174,32 @@ class Generator:
         # omitted args.
         return (<object>y).astype(dtype, copy=False)
 
+    def exponential(self, scale=1.0, size=None, dtype=numpy.float64):
```

Comment:
Exponential seems not to have dtype
https://numpy.org/doc/stable/reference/random/generated/numpy.random.Generator.exponential.html

Reply:
I added this because currently beta distribution and is having dtype in
https://github.com/cupy/cupy/blob/master/cupy/random/_generator_api.pyx#L145-L146
but not in 
https://github.com/numpy/numpy/blob/60cd9d717524e5a003bee9e4270b9c6b8144a7af/numpy/random/_generator.pyx#L294

Should we remove that also?

Reply:
yes, that is my mistake 😂
lets do it in another pr

Reply:
So for exponential, I am changing it in this pr. And for beta will make another.